### PR TITLE
skip rather than report error while running `ninja test` when libfuse is subproject

### PR DESCRIPTION
**Pull request by ahmed-masud**
Have wrong_command return a skip code (77) rather than get an error when libfuse is a subproject

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -65,7 +65,7 @@ configure_file(output: 'config.h',
 #
 add_project_arguments('-D_REENTRANT', '-DHAVE_CONFIG_H', '-Wall', '-Wextra', '-Wno-sign-compare',
                       '-Wstrict-prototypes', '-Wmissing-declarations', '-Wwrite-strings',
-                      '-fno-strict-aliasing', language: 'c')
+                      '-fno-strict-aliasing', meson.is_subproject()? '-DMESON_IS_SUBPROJECT' : '-UMESON_IS_SUBPROJECT', language: 'c')
```
Comment (Nikratio) [inline meson.build]: This is hard to read. Please make it

```
if meson.is_subproject():
   add_project_arguments('-DIS_MESON_SUB...')
```

File: test/wrong_command.c [hunk 2]
```diff
@@ -5,5 +5,9 @@ int main(void) {
 		"This is not the command you are looking for.\n"
 		"You probably want to run 'python3 -m pytest test/' instead"
 		"\e[0m\n");
+#if defined(MESON_IS_SUBPROJECT)
```
Comment (Nikratio) [inline test/wrong_command.c]: Why not `#ifdef`?

Comment (ahmed-masud) [reply test/wrong_command.c]: It's a force of habit :) I can fix that too. let me close this pull request make al the changes and see if I can do it more appropriately :)

**Comment by Nikratio** [comment]
You know that you can just change the pull request, right?

**Pull request closed without merging**